functions/mqhandlers.go

```diff
@@ -224,12 +224,16 @@ func HostUpdate(client mqtt.Client, msg mqtt.Message) {
 		}
 		restartDaemon = true
 	case models.DeleteHost:
-		clearRetainedMsg(client, msg.Topic())
-		unsubscribeHost(client, serverName)
-		deleteHostCfg(client, serverName)
-		config.WriteNodeConfig()
-		config.WriteServerConfig()
-		resetInterface = true
+		faults, err := LeaveNetwork(hostUpdate.Node.Network, true)
```

Comment:
why are we leaving the network here on the delete host prompt?

Reply:
So the server will get the message and delete the zombie node

Reply:
doesn't that happen when you delete a node already?